release-notes/release-notes57.md

```diff
@@ -37,7 +37,17 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
-### Enhancement name
+### The `spellchecker_ignorelist` option now accepts arrays of words per language
+
+{{site.productname}} 5.7 introduces enhancements to the `spellchecker_ignorelist` (formerly `spellchecker_whitelist`, see: [The `spellchecker_whitelist` option has been renamed](#thespellchecker_whitelistoptionhasbeenrenamed)).
```

Comment:
```suggestion
{{site.productname}} 5.7 introduces enhancements to the `spellchecker_ignorelist` (formerly `spellchecker_whitelist`. See: [The `spellchecker_whitelist` option has been renamed](#thespellchecker_whitelistoptionhasbeenrenamed)).
```

Reply:
@tylerkelly13 I've seen both ", see" and ". See" - which would you prefer here?

Reply:
normally we use `, see` in the context of `For information on the <thing(s)>, see:`